File: api/examples/c#/BUBBLE_SAVED.cs
using System;
using System.Collections.Generic;

[Serializable()]
public partial class BUBBLE_SAVED
{
    public long ID = 0L;
    public int IsInstance = 0;
    public string Name = string.Empty;
    public string ProfilePath = string.Empty;
    public long BrowserID = 0L;
    public long DNSID = 0L;
    public List<long> ProxyList = new List<long>();
    public long VPNID = 0L;
    public LAUNCH_ACTIONS Launch = new LAUNCH_ACTIONS();
    public int Options = 0;
    public int BorderColor = 0;
    public Dictionary<int, string> SpoofSettings = new Dictionary<int, string>();
}

File: api/examples/vb.net/BUBBLE_SAVED.vb
<Serializable()>
Public Class BUBBLE_SAVED
    Public ID As Long = 0
    Public IsInstance As Integer = 0
    Public Name As String = String.Empty
    Public ProfilePath As String = String.Empty
    Public BrowserID As Long = 0
    Public DNSID As Long = 0
    Public ProxyList As New List(Of Long)
    Public VPNID As Long = 0
    Public Launch As New LAUNCH_ACTIONS
    Public Options As Integer = 0
    Public BorderColor As Integer = 0
    Public SpoofSettings As New Dictionary(Of Integer, String)
End Class
